### PR TITLE
feat: suggestions from vicky

### PR DESCRIPTION
**Pull request by sreekanth221998**
## Summary by Sourcery

Documentation:
- Update captions in figures and tables to reflect the correct measurement of model performance as the fraction of questions answered correctly, rather than completely correctly.

### CODE REVIEW

File: src/tex/appendix.tex [hunk 1]
```diff
@@ -374,12 +374,12 @@ \subsection{Human baseline} \label{sec:human_baseline}
...
     \includegraphics{figures/human_score_distribution.pdf}
     \script{plot_human_score_distribution.py}
-    \caption{\textbf{Distribution of human scores.} The histogram and kernel density estimates show the fraction of questions answered completely correctly.
+    \caption{\textbf{Distribution of human scores.} The histogram and kernel density estimates show the fraction of questions answered correctly.
     Since the best possible score for each question is one and the worst possible score is zero, the values on this plot are between zero and one.}
     \label{fig:human_score_distribution}
 \end{figure}
 
-We also recorded the time humans took to answer the questions. This time is the time from the question being displayed to the human to the human submitting the answer.
+We also recorded the time humans took to answer the questions \cref{fig:human_timing}. This time is the time from the question being displayed to the human to the human submitting the answer.
```
Comment (kjappelbaum) [inline src/tex/appendix.tex:382]:
```suggestion
We also recorded the time humans took to answer the questions \Cref{fig:human_timing}. This time is the time from the question being displayed to the human to the human submitting the answer.
```

File: src/tex/ms.tex [hunk 2]
```diff
@@ -81,7 +81,7 @@ \section{Results and Discussion}
     The framework's foundation is the benchmark corpus that consists of many questions and answers that we manually or semi-automatically compiled from various sources.
     Questions are classified based on topics, required skills (reasoning, calculation, knowledge, intuition), and difficulty levels.
     We then used this corpus to evaluate the performance of various models and tool-augmented systems using a custom framework. To provide a baseline, we built a web application that we used to survey experts in chemistry.
-    The results of the evaluations are then compiled in publicly accessible leaderboards, which we propose as a foundation for evaluating future models.
+    The results of the evaluations are then compiled in publicly accessible leaderboards \ref{sec:leaderboard}, which we propose as a foundation for evaluating future models.
```
Comment (kjappelbaum) [inline src/tex/ms.tex:84]:
```suggestion
    The results of the evaluations are then compiled in publicly accessible leaderboards (\Cref{sec:leaderboard}), which we propose as a foundation for evaluating future models.
```

File: src/tex/ms.tex [hunk 3]
```diff
@@ -153,9 +153,9 @@ \subsection{Model evaluation}
...
 
 Notably, we find that models are still limited in their ability to answer knowledge-intensive questions (\Cref{tab:performance_table_human_subset}), that is they did not memorize the relevant facts. Our results indicate that this is not a limitation that could be overcome by simple application of \gls{rag} systems such as \PaperQATwo. This is likely because the required knowledge cannot easily be accessed via papers (which is the only external knowledge \PaperQATwo has access to) but rather by lookup in specialized databases (e.g., PubChem, Gestis), which also the humans in our study used to answer such questions (\Cref{fig:tool_use}).
-This indicates that there is still room for improvement of chemical \gls{llm} by training them on more specialized data sources or by integrating them with specialized databases.
+This indicates that there is still room for improvement in chemical \gls{llm}s by training them on more specialized data sources or by integrating them with specialized databases.
```
Comment (kjappelbaum) [inline src/tex/ms.tex:156]:
```suggestion
This indicates that there is still room for improvement in chemical \glspl{llm} by training them on more specialized data sources or by integrating them with specialized databases.
```

File: src/tex/ms.tex [hunk 4]
```diff
@@ -153,9 +153,9 @@ \subsection{Model evaluation}
...
 
-In addition, our analysis shows that the performance of models is correlated with their size (see \Cref{fig:model_size_plot}). This is in line with observations in other domains but also indicates that chemical \gls{llm} could, to some extent, be further improved by scaling them up.
+In addition, our analysis shows that the performance of models is correlated with their size (see \Cref{fig:model_size_plot}). This is in line with observations in other domains but also indicates that chemical \gls{llm}s could, to some extent, be further improved by scaling them up.
```
Comment (kjappelbaum) [inline src/tex/ms.tex:158]:
```suggestion
In addition, our analysis shows that the performance of models is correlated with their size (see \Cref{fig:model_size_plot}). This is in line with observations in other domains but also indicates that chemical \glspl{llm} could, to some extent, be further improved by scaling them up.
```

File: src/tex/references.bib [hunk 5]
```diff
@@ -1405,3 +1405,14 @@ @book{bloom1956taxonomy
   year={1956},
   publisher={Longmans, Green}
 }
+
+@article{wu2018moleculenet,
```
Comment (kjappelbaum) [inline src/tex/references.bib:1409]: and this probably already is cited somewhere

File: src/tex/ms.tex [hunk 6]
```diff
@@ -47,7 +47,7 @@ \section{Introduction}
 
 However, the rapid increase in capabilities of chemical \gls{ml} models led (even before the recent interest in \glspl{llm}) to concerns about the potential for dual use of these technologies, e.g., for the design of chemical weapons.\autocite{gopal2023releasing, ganguli2022red, Urbina_2022, campbell2023censoring, moulange2023towards, urbina2022teachable}
 To some extent, this is not surprising as any technology that, for instance, is used to design non-toxic molecules can also be used inversely to predict toxic ones (even though the synthesis would still require access to controlled physical resources and facilities).
-Still, it is essential to realize that such models' user base is wider than that of chemistry and materials science experts who can critically reflect on every output such models produce.
+Still, it is essential to realize that LLMs' user base is wider than that of chemistry and materials science experts who can critically reflect on every output these models produce.
```
Comment (kjappelbaum) [inline src/tex/ms.tex:50]:
```suggestion
Still, it is essential to realize that the user base of \glspl{llm} is broader than that of chemistry and materials science experts who can critically reflect on every output these models produce.
```

**Comment by kjappelbaum** [comment]
@sreekanth221998 could you perhaps also fix the merge conflicts?

**Comment by sreekanth221998** [comment]
> @sreekanth221998 could you perhaps also fix the merge conflicts?

working on it Kevin, 
Thanks

File: src/tex/appendix.tex [hunk 7]
```diff
@@ -344,6 +360,7 @@ \subsection{Human baseline} \label{sec:human_baseline}
 The application is hosted on the Vercel web hosting platform.
 
 In the applications, human participants were presented with molecules as rendered drawings and SMILES strings. \LaTeX\xspace equations and chemical equations were rendered using MathJax (\Cref{fig:screenshots}).
+In the applications, human participants were presented with molecules as rendered drawings and SMILES strings. \LaTeX\xspace equations and chemical equations were rendered using MathJax (\Cref{fig:screenshots}).
```
Comment (kjappelbaum) [inline src/tex/appendix.tex:363]:
```suggestion
```

File: src/tex/ms.tex [hunk 8]
```diff
@@ -163,9 +163,9 @@ \subsection{Model evaluation}
...
+    \caption{\textbf{Performance of the models and humans on the different topics of the \enquote{tiny} subset of the \chembench corpus.} The radar plot shows the performance of the models and humans on the different topics of \enquote{tiny} subset of the \chembench corpus. The performance is measured as the fraction of questions that were answered correctly by the models.
     The best score for every dimension is one (all questions answered correctly), and the worst is zero (no question answered correctly). A larger colored area indicates a better performance.
-    This figure shows the performance on \chembenchmini. The performance of models on the entire corpus is shown in \Cref{fig:performance_per_topic}.
+    This figure shows the performance on the subset of questions that were answered by humans. The performance of models on the entire corpus is shown in \Cref{fig:barplot_all_correct_all_questions}.
```
Comment (kjappelbaum) [inline src/tex/ms.tex:168]: I think the `\chembenchmini` version is the one we agreed on

File: src/tex/ms.tex [hunk 9]
```diff
@@ -151,9 +151,9 @@ \subsection{Model evaluation}
...
 
 Notably, we find that models are still limited in their ability to answer knowledge-intensive questions (\Cref{tab:performance_table_human_subset}), that is they did not memorize the relevant facts. Our results indicate that this is not a limitation that could be overcome by simple application of \gls{rag} systems such as \PaperQATwo. This is likely because the required knowledge cannot easily be accessed via papers (which is the only external knowledge \PaperQATwo has access to) but rather by lookup in specialized databases (e.g., PubChem, Gestis), which also the humans in our study used to answer such questions (\Cref{fig:tool_use}).
-This indicates that there is still room for improvement of chemical \gls{llm} by training them on more specialized data sources or by integrating them with specialized databases.
+This indicates that there is still room for improvement in chemical \glspl{llm} by training them on more specialized data sources or by integrating them with specialized databases.
```
Comment (kjappelbaum) [inline src/tex/ms.tex:154]: why the switch from of -> in

File: src/tex/appendix.tex [hunk 10]
```diff
@@ -215,7 +231,7 @@ \subsection{Model performance} \label{sec:model_performance_app}
...
+    \caption{\textbf{Fraction of completely correctly answered questions per data source on the \enquote{tiny} subset.} The heatmap shows, in color, the fraction of questions answered correctly by different systems for some of our data sources. The performance is measured as the fraction of questions answered correctly by the models. A score of one (red) indicates that all questions were answered correctly, while a score of zero (blue) indicates that none were answered correctly.
         We see that the performance of the models varies significantly between the different data sources. For instance, it is interesting to observe that questions sourced based on textbooks seem easier for the leading models than for humans. However, this performance does not correlate with performance on other sources, e.g., semi-programmatically created tasks such as questions about the number of signals in an \gls{nmr} spectrum.
```
Comment (kjappelbaum) [inline src/tex/appendix.tex:235]: the `\chembenchmini` version should be the one we take

File: src/tex/appendix.tex [hunk 11]
```diff
@@ -166,20 +166,36 @@ \subsection{Model performance} \label{sec:model_performance_app}
...
+    \label{tab:performance_table_human_subset}
+\end{table}
 
 \begin{figure}[htb]
     \centering
     \includegraphics[width=\textwidth]{figures/all_questions_models_completely_correct_radar_overall.pdf}
-    \caption{\textbf{Performance of the models on the different topics of the \chembench corpus.} The radar plot shows the performance of the models on the different topics of the \chembench corpus. The performance is measured as the fraction of questions answered completely correctly by the models.
-    A score of 1 (full coverage until the outer line of this plot) indicates that all questions were answered completely correctly, while a score of 0 indicates that none were answered completely correctly.
+    \caption{\textbf{Performance of the models on the different topics of the \chembench corpus.} The radar plot shows the performance of the models on the different topics of the \chembench corpus. The performance is measured as the fraction of questions answered correctly by the models.
+    A score of 1 indicates that all questions were answered correctly, while a score of 0 indicates that none were answered correctly.
```
Comment (kjappelbaum) [inline src/tex/appendix.tex:198]: the "full coverage until the outer line" was feedback from Santiago

File: src/tex/appendix.tex [hunk 12]
```diff
@@ -166,20 +166,36 @@ \subsection{Model performance} \label{sec:model_performance_app}
...
+    \label{tab:performance_table}
+\end{table}
+
+\begin{table}
+    \caption{\textbf{Performance of the models on the \enquote{tiny} subset of the \chembench corpus.} The table shows the fraction of questions answered correctly by the models for different skills and difficulty levels.}
+    \resizebox{\textwidth}{!}{
+    \variable{output/performance_table_human_subset.tex}
+    }
+    \label{tab:performance_table_human_subset}
+\end{table}
```
Comment (kjappelbaum) [inline src/tex/appendix.tex:192]: here, also the `tiny` stuff should be `\chembenchmini` as it currently is on main

File: src/tex/appendix.tex [hunk 13]
```diff
@@ -365,14 +382,12 @@ \subsection{Human baseline} \label{sec:human_baseline}
     \centering
     \includegraphics{figures/human_score_distribution.pdf}
     \script{plot_human_score_distribution.py}
-    \caption{\textbf{Distribution of human scores.} The histogram and kernel density estimates show the fraction of questions answered correctly by the human volunteers.
-    Since the best possible score for each question is one and the worst possible score is zero, the values on this plot are between zero and one. A score of one would mean that a volunteer answered all questions correctly. A score of zero would mean that no question was answered correctly.
-    We find that the scores for the questions that the human volunteers answered with tools are generally lower than the scores for the questions that the human volunteers answered without tools.
-    }
+    \caption{\textbf{Distribution of human scores.} The histogram and kernel density estimates show the fraction of questions answered correctly.
```
Comment (kjappelbaum) [inline src/tex/appendix.tex:385]: also in this caption there is more explanation in the version on `main` right now, which we should keep

**Comment by kjappelbaum** [comment]
I think the merging made some of the changes we made in `main` go away in your version

**Comment by sreekanth221998** [comment]
> I think the merging made some of the changes we made in `main` go away in your version

may be will revert this branch

**Comment by kjappelbaum** [comment]
> > I think the merging made some of the changes we made in `main` go away in your version
> 
> may be will revert this branch

do you already have a suggestion on how we should go about this?

**Comment by kjappelbaum** [comment]
@sreekanth221998 will you take care of updating things or shall I do it?

**Comment by sreekanth221998** [comment]
> @sreekanth221998 will you take care of updating things or shall I do it?

i wanted to do it but i don't know how to go on this.

I will revert to this commit  [fix suggestions from kevin](https://github.com/lamalab-org/chembench-paper/pull/68/commits/4eb460b509d7b664a20f6dcc0e10a045b72518df) and we merge.

**Comment by kjappelbaum** [comment]
> > @sreekanth221998 will you take care of updating things or shall I do it?
> 
> i wanted to do it but i don't know how to go on this.
> 
> I will revert to this commit [fix suggestions from kevin](https://github.com/lamalab-org/chembench-paper/pull/68/commits/4eb460b509d7b664a20f6dcc0e10a045b72518df) and we merge.

A simply way is that you can also just in the current state of this PR go through your changes and then verify that we use `\chembenchmini` etc. If you look in the diff view on GitHub, this should be doable. As you see on the left, the current state is `main`, and on the right, your version. And you want to align those.

**Comment by sreekanth221998** [comment]
> > > @sreekanth221998 will you take care of updating things or shall I do it?
> > 
> > 
> > i wanted to do it but i don't know how to go on this.
> > I will revert to this commit [fix suggestions from kevin](https://github.com/lamalab-org/chembench-paper/pull/68/commits/4eb460b509d7b664a20f6dcc0e10a045b72518df) and we merge.
> 
> A simply way is that you can also just in the current state of this PR go through your changes and then verify that we use `\chembenchmini` etc. If you look in the diff view on GitHub, this should be doable. As you see on the left, the current state is `main`, and on the right, your version. And you want to align those.

Thanks Kevin, i will fix it

File: src/tex/appendix.tex [hunk 14]
```diff
@@ -175,11 +175,27 @@ \subsection{Model performance} \label{sec:model_performance_app}
...
+\begin{table}
+    \caption{\textbf{Performance of the models on the \chembench corpus.} The table shows the fraction of questions answered correctly by the models for different skills and difficulty levels.}
+    \resizebox{\textwidth}{!}{
+    \variable{output/performance_table.tex}
+    }
+    \label{tab:performance_table}
+\end{table}
+
+\begin{table}
+    \caption{\textbf{Performance of the models on the \enquote{tiny} subset of the \chembench corpus.} The table shows the fraction of questions answered correctly by the models for different skills and difficulty levels.}
```
Comment (kjappelbaum) [inline src/tex/appendix.tex:187]: it is not tiny but `\chembenchmini`

Comment (kjappelbaum) [inline src/tex/appendix.tex:187]:
```suggestion
    \caption{\textbf{Performance of the models on \chembenchmini.} The table shows the fraction of questions answered correctly by the models for different skills and difficulty levels.}
```

**Review by kjappelbaum — APPROVED** [review]

**Pull request merged**